draft-ietf-quic-manageability.md

```diff
@@ -287,10 +287,14 @@ Hello, and the server can send 1-RTT data as soon as it has sent its Server
 Hello. The Client Completion flight contains at least one Handshake packet and
 could also include an Initial packet. QUIC packets in separate contexts during
 the handshake can be coalesced (see {{coalesced}}) in order to reduce the
-number of UDP datagrams sent during the handshake.  QUIC packets can be lost
-and reordered, so packets within a flight might not be sent close in time,
-though the sequence of the flights will not change, because one flight depends
-upon the peer's previous flight.
+number of UDP datagrams sent during the handshake.
+
+Handshake packets can arrive out-of-order without impacting the handshake as
+long as the reordering did not cause extensive delays which would be consider
+as loss by either side. If QUIC packets get lost or reordered, packets belonging
```

Comment:
I think "did not cause extensive delays which would be consider as loss by either side." may be unclear.

In order to declare a packet lost, a later packet needs to be acknowledged, so it's more complex than time passing.  I would be inclined to remove this clause.
```suggestion
If QUIC packets get lost or reordered, packets belonging
```

Reply:
There are multiple cases:

1) The client never gets a reply from the server and must assume that all its initial packets got lost. I actually was briefly looking in the spec and didn't find quickly any information about a handshake retry time-out...?

2) One packet of the handshake is lost but others are acknowledged. This usually needs multiple ACK to detect loss, right?

3) Similar as tail loss, loss is detect due to a timeout. I thought that is the most likely case for loss in the handshake no?

I added this based on Al's comment in the opsdir, asking when/if reorder can impact the handshake negatively. Would be nice to say something about this, as I can definitely see who this question is interesting for network operator.

Reply:
1. There's no handshake retry time-out in the specs.  I expect most clients have one in practice.
2. This is complex, you only need an ACK of one packet sent afterwards, but it has to be in the same PN space, so commonly you end up relying on the PTO timer.
3. Yes, this is the most likely case.  When the timer goes off, the packet is not actually declared lost, which is an important distinction, at least in my thinking.

Instead of loss, maybe "did not cause extensive delays that trigger a spurious Probe Timeout (Section 6.2 of RFC9002)."?

Reply:
Ah got it! It's about not calling it loss as it does doesn't trigger normal loss detect. Will make that edit!